src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -743,6 +743,20 @@ def has_pending_clones(self, snapname):
                 return False
             raise
 
+    def remove_non_existent_pending_clones(self, path, link_path, track_id):
+        try:
+            self.fs.lstat(link_path)
+        except cephfs.Error as e:
+            if e.args[0] == errno.ENOENT:
+                try:
+                    log.info(f"Cleaning up dangling symlink for the clone: {path}")
+                    self.fs.unlink(path)
+                    self._remove_snap_clone(track_id)
+                except (cephfs.Error, MetadataMgrException) as e:
+                    log.warning(f'Removing of dangling symlink for the clone {path}'
+                                f'failed with the exception:"{e}"')
```

Comment:
I think I did leave a comment about introducing a space so that "failed" does not clobber $path logged just before.

Reply:
+1

Reply:
Added a space before `failed`